### DISCUSSION TITLE
Question 47: We discovered a possible issue with the catalog #s when turning records over from Exhibitions to Anthro [MG]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#593] [Q&A] [answered]
Description: Exhibition catalogue #’s remain after changing to Anthro and they use a different catalog number and fields.

Unique number/part-number restricts fixing for Anthro if want to use same part# (less likely for Zoo/Bot; not issue for Geo since uses same field as Exh)

Cat IRNs for attempted/temporary fixes:  

| Cat# | SummaryData | irn |
| --- | --- | --- |
| NAT-578-B | Nanibaa Beck moccasin button (for left moc - Dinetah) | 4335193 |
| NAT-578-C | Nanibaa Beck moccasin button (for left moc - Kayah) | 4335204 |
| NAT-578-B, NAT-578-C | Nanibaa Beck moccasin buttons (pair for left moc) | 4423557 |
| NAT-579-B | Nanibaa Beck moccasin button (for right moc - Dinetah) | 4423604 |
| NAT-579-C | Nanibaa Beck moccasin button (for right moc - Kayah) | 4423605 |
| NAT-579-B, NAT-579-C | Nanibaa Beck moccasin buttons (pair for right moc) |   |

**Comment by rondlg** [comment]
Response: Anthro and Exhibition Catalog number fields are not the same. Switch on the catalogue Tab that Exhibition uses for Anthropology.

**Action for IT:** Need to make the Exhibitions catalog tab available for Anthro so that they can reconcile parts field with the sub# field that they use when records are turned back over to them.

**Comment by rondlg** [comment, marked as answer]
The AllCatGTab is now visible for the AnthroMgr group and they should see the Exhibition catalogue number fields.

**Reply by ElanaDux** [reply]
I see it now.   Should I clear-out the GeoCatalogNo and GeoPartNo fields when the records are turned over to Anthro?

**Comment by rondlg** [comment]
You can Just make sure that the data goes over into the "Other Numbers" field on that tab so that we can check back later if needs be.

**Reply by ElanaDux** [reply]
Gotchya
